features/dryrun_mode.feature

```diff
@@ -0,0 +1,134 @@
+Feature: Dryrun mode
+
+  Using the `--dry-run` or `-d` flag gives you a way to quickly scan your features without actually running them.
+
+  Background:
+    Given a file named "features/a.feature" with:
+    """
+    Feature: Missing
+      Scenario: Missing
+        Given this step passes
+    """
+
+  Scenario: Skipped scenario with implemented step with --dry-run
+    Given a file named "features/step_definitions/cucumber_steps.js" with:
+    """
+    var cucumberSteps = function() {
+      this.When(/^this step passes$/, function(callback) { callback(); });
+    };
+    module.exports = cucumberSteps;
+    """
+    When I run cucumber.js with `-f progress features/a.feature --dry-run`
+    Then it outputs this text:
+    """
+    -
+
+    1 scenario (1 skipped)
+    1 step (1 skipped)
+
+    """
+    And the exit status should be 0
+
+  Scenario: Skipped scenario with passing step with --dry-run --strict
```

Comment:
can you please use `defined` instead of `implemented` or `passing` when describing the step?